jobs/ml-ops/s3/data-store/main.py

```diff
@@ -0,0 +1,26 @@
+from dotenv import load_dotenv
+import os, sys, boto3
+
+def main() -> int:
+    """
+    This function uploads a local CSV file to a target S3 bucket
+    """
+
+    load_dotenv(dotenv_path='./.env')
+
+    s3 = boto3.resource(
+        's3',
+        region_name=os.getenv('SCW_REGION'),
+        use_ssl=True,
+        endpoint_url='https://s3.{region}.scw.cloud'.format(region=os.getenv('SCW_REGION')),
+        aws_access_key_id=os.getenv('SCW_ACCESS_KEY'),
+        aws_secret_access_key=os.getenv('SCW_SECRET_KEY'),
+    )
+
+    bucket = s3.Bucket(name=os.getenv('SCW_DATA_STORE')) # type: ignore
```

Comment:
Sorry I'm annoying ^^ But typically this is why I use `os.environ["SCW_DATA_STORE"]`, I prefer crashing the application if a required env var is not defined and it simplifies typing (not need to deal with the `None` case)

Reply:
Also use double quotes not single quotes everywhere (I think Black should enforce this)